roles/chameleon_usage/templates/chameleon-usage-extract-data.j2

```diff
@@ -3,9 +3,12 @@
 docker run --rm --net=host \
     -v "{{ backup_dir }}:{{ backup_dir }}" \
     -v "{{ chameleon_usage_config_dir }}/my.cnf:/etc/mysql/my.cnf" \
+    -v "{{ chameleon_usage_config_dir }}/site.yaml:/etc/chameleon_usage/site.yaml" \
     -v "/var/run/docker.sock:/var/run/docker.sock" \
     {{ chameleon_usage_docker_image }} \
-    {{ usage_extract_command }} --site {{chameleon_site_name}} --backup-location {{ backup_dir }} \
+    {{ usage_extract_command }} --site {{openstack_region_name}} \
```

Comment:
Why not keep it `chameleon_site_name`? Only reason is, it may not be expected that changing the region name will change/break usage reporting. In practice we depend on region name in a lot of places so I think that's OK, but I'm thinking that maybe in the future we will want to have `chameleon_site_name` be more like `chameleon_site_id`? Or something. Like, we have `uc` as the ID and the display name is `CHI@UC`. cc @msherman64

Reply:
The outage tags are using region names, so I thought it would be easier to just use region names for everything else instead of changing back and forth. But if we prefer the site name, I'll make changes to the usage reporting scripts accordingly.

Reply:
IDK, @msherman64 what do you think? I don't have strong feelings I guess. Good point about outage tags using the region name currently @zhenz .